src/test/java/LoginPageTest.java

```diff
@@ -0,0 +1,43 @@
+import config_loaders.EnvironmentConfigLoader;
+import config_loaders.UserConfigLoader;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import pages.LoginPage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class LoginPageTest {
+    private LoginPage loginPage;
+    private static WebDriver driver;
+    private static EnvironmentConfigLoader environmentConfigLoader = new EnvironmentConfigLoader();
+    private UserConfigLoader userConfigLoader = new UserConfigLoader();
+
+    @Test
+    void checkIfLoginDefaultUser() {
+        loginPage.userName().sendKeys(userConfigLoader.getUserLogin());
```

Comment:
These actions/checks must be hidden under PageObject. So your LoginPage should have methods 'setLogin()', 'setPass()', 'loginWithCredentials()', etc.

Reply:
https://martinfowler.com/bliki/PageObject.html